services/movie.py

```diff
@@ -0,0 +1,35 @@
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> Movie:
```

Comment:
The return type hint for `get_movies` should be `QuerySet` instead of `Movie` because the function returns a QuerySet of `Movie` instances, not a single instance.